src/main/java/com/fundation/webservice/controller/StorageService.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.UrlResource;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+/**
+ * Implements a Spring service that handles copying to the upload directory.
+ *
+ * @author Alejandro S�nchez Luizaga
+ * @version 1.0
+ */
+@Service
+public class StorageService {
+
```

Comment:
Is it necessary this space?

Reply:
Solved

Reply:
The extra line has now been removed. Somehow I had forgotten to push the changes. Sorry